src/components/clusterConfiguration/DiscoveryImageForm.tsx

```diff
@@ -133,7 +133,7 @@ const DiscoveryImageForm: React.FC<DiscoveryImageFormProps> = ({
                 <TextAreaField
                   label="SSH public key"
```

Comment:
I think 'installation' is misleading as that term is used for cluster installation rather than host registration across the application. 'Installation SSH Key' I'd expect to be a key provided for the cluster installation rather than host registration. How about 'Discovery Image SSH Public Key'?

Reply:
Alternatively we could keep this one intact 'SSH public key' and in the switch refer to this as 'Host discovery SSH key'

Reply:
The `Discovery Image SSH Public Key` is clear from user perspective since it points where it is entered.

Anyway I don't think it is clear what are these two keys used for and especially why the user might want to make them different.

What about `Host SSH Public Key while instlling` and `Host SSH Public Key for troubleshooting after installation`?

Reply:
> The `Discovery Image SSH Public Key` is clear from user perspective since it points where it is entered.
> 

I would challenge that assumption, you are assuming the user understand what is about to happen...


perhaps we can have the setting of reusing the same key where the user first set it (e.g. in the iso creation phase)? and then have a checkbox for using different ssh keys for the cluster and troubleshooting installation.

IMHO 99/100 folks would always use the same ssh key.